packages/ponder-subgraph-api/src/graphql.ts

```diff
@@ -584,6 +600,48 @@ export function buildGraphQLSchema(
       });
     });
 
+  // Subgraph Meta Field
+  queryFields._meta = {
+    type: new GraphQLObjectType({
+      name: "_Meta_",
+      fields: {
+        block: {
+          type: new GraphQLNonNull(
+            new GraphQLObjectType({
+              name: "_Block_",
+              fields: {
+                hash: { type: GraphQLString },
+                number: { type: new GraphQLNonNull(GraphQLInt) },
+                timestamp: { type: new GraphQLNonNull(GraphQLInt) },
+                parentHash: { type: GraphQLString },
+              },
+            }),
+          ),
+        },
+        deployment: { type: new GraphQLNonNull(GraphQLString) },
+        hasIndexingErrors: { type: new GraphQLNonNull(GraphQLBoolean) },
+      },
+    }),
+    resolve: async (_source, _args, context) => {
+      const status = await queryPonderStatus(metaConfig.schema, context.drizzle);
+      const mainnetStatus = status.find((status) => status.network_name === "1");
```

Comment:
Does that mean we only need to support metadata for the mainnet ENS Deployment, and we don't need support for Sepolia or Holesky deployments?